src/language-server/simple-ui.langium

```diff
@@ -50,10 +50,15 @@ NumberExpression:
 SymbolReference:
     symbol=[Parameter];
 
+// Implementation of the <section> element
+
+Section: 
+    'section' name=ID? description=STRING? classes=CSSClasses? content+=BodyElement* 'end';
```

Comment:
Maybe like we did in components: 
```
'section' name=ID? description=STRING? classes=CSSClasses? '{' content+=BodyElement* '}';
```

Reply:
I agree with you but I will not change it for this PR.

The grammar will be refactored (very) soon and I'll make sure to check that the syntax is consistent between elements.